src/conference.c

```diff
@@ -223,17 +225,32 @@ int64_t init_conference_win(Toxic *toxic, uint32_t conferencenum, uint8_t type,
             // probably it so happens that this will (at least typically) be
             // the case, because toxic and tox maintain the indices in
             // parallel ways. But it isn't guaranteed by the API.
```

Comment:
Is this comment still attached to the right statement?

Reply:
It's kind of randomly placed in the first place. We don't actually set the conferencenumber to the index anywhere in that code block. It is assumed other place in the code though which does need to be fixed eventually.